### DISCUSSION TITLE
What is key called pathname in <Image>?

### DISCUSSION THREAD

**Discussion by 96312** [vercel/next.js#61331] [Help] [answered]
### Summary

Hello. I'm trying to link an external image following this page

- https://nextjs.org/docs/messages/next-image-unconfigured-host
- Image is = https://images.pexels.com/photos/39501/lamborghini-brno-racing-car-automobiles-39501.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1

What's the dummy `/account123/` and what should be in it's place?

### Additional information

```js
images: {
    remotePatterns: [
      {
        protocol: 'https',
        hostname: 'images.pexels.com',
        port: '',
        pathname: '/account123/**',
      },
    ],
  },
```

**Comment by makibat** [comment, marked as answer]
The example that you've provided ensures that the next/image's `src` property must start with `https://images.pexels.com/account123/`

If you're unsure of what the exact `pathname` to your choice of external images is, you can just leave it out of your config and it should work just fine:
``` js
  images: {
    remotePatterns: [
      {
        protocol: 'https',
        hostname: 'images.pexels.com',
        port: ''
      }
    ]
  },
```
